csharp/Platform.Ranges/Range[T].cs

```diff
@@ -19,22 +19,25 @@ public struct Range<T> : IEquatable<Range<T>>
         private static readonly EqualityComparer<T> _equalityComparer = EqualityComparer<T>.Default;
 
         /// <summary>
-        /// <para>Returns minimum value of the range.</para>
-        /// <para>Возвращает минимальное значение диапазона.</para>
+        /// <para>Provides a minimum value of the range.</para>
+        /// <para>Представляет минимальное значение диапазона.</para>
         /// </summary>
         public readonly T Minimum;
 
         /// <summary>
-        /// <para>Returns maximum value of the range.</para>
-        /// <para>Возвращает максимальное значение диапазона.</para>
+        /// <para>Provides a maximum value of the range.</para>
+        /// <para>Представляет максимальное значение диапазона.</para>
         /// </summary>
         public readonly T Maximum;
 
         /// <summary>
-        /// <para>Initializes a new instance of the Range class.</para>
-        /// <para>Инициализирует новый экземпляр класса Range.</para>
+        /// <para>Initializes a new instance of the <see cref="Range{T}"/> structure with the specified minimum and maximum values.</para>
```

Comment:
```suggestion
        /// <para>Initializes a new instance of the <see cref="Range{T}"/> structure with a single specified value as minimum and maximum values.</para>
```

`values` is a plural word.